src/components/container/DailyDataGoal/DailyDataGoal.tsx

```diff
@@ -33,13 +33,13 @@ export default function (props: DailyDataGoalProps) {
     const date = props.date ?? dateRangeContext?.intervalStart ?? startOfDay(new Date());
 
     useInitializeView(() => {
-        const startDate = add(date, { days: -1 });
-        const endDate = add(date, { days: 2 });
+        const startDate = startOfDay(date);
+        const endDate = endOfDay(date);
```

Comment:
I'm curious - was this actually causing misbehaviors?  I'm pretty sure that, somewhere a couple of layers below this in the actual queries we already "pad" the date range requested for a variety of timezone/sync related reasons, but this code still only looks at the result of a single day.  So was the only "bad effect" of this that we're getting a larger data set back than necessary?

Reply:
This is intended for looking at one day of data, what is the value of padding extra days back/fwd? Is there a use case or something with the returned data that isn't apparent here?

Reply:
I wasn't suggesting changing it back, I was asking if it was *actually* causing a participant facing bad behavior, or just extra data load.  As I said, in the underlying code it buffers the timestamps slightly to ensure you actually get the data you're looking for, so this double buffering certainly isn't necessary.  I was just curious.

Reply:
No, it wasn't showing the wrong data. I came across it while troubleshooting and found it misleading